### DISCUSSION TITLE
Allow to display custom length of a field on listing

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#2980] [Archive]
Currently, I see this:

<img width="923" alt="Screen Shot 2020-11-12 at 18 56 16" src="https://user-images.githubusercontent.com/4686040/98970646-d94bd300-2518-11eb-9004-f5cad377bd8f.png">

Can we allow to see more from that string?

**Comment by rijkvanzanten** [comment]
What display is this? How wide is that column? You can resize the columns in the table

**Reply by botzill** [reply]
Yes you right, I didn't notice that we can resize columns. Thx, now is OK.

**Reply by benhaynes** [reply]
I would like to have displays/fields be able to set min/max widths... but maybe this isn't needed. 🤔
